kolibri/logger/models.py

```diff
@@ -128,9 +132,23 @@ class UserSessionLog(BaseLogModel):
     user = models.ForeignKey(FacilityUser)
     channels = models.TextField(blank=True)
     start_timestamp = models.DateTimeField(auto_now_add=True)
-    completion_timestamp = models.DateTimeField(blank=True, null=True)
+    last_interaction_timestamp = models.DateTimeField(auto_now=True, null=True)
     pages = models.TextField(blank=True)
 
+    @classmethod
```

Comment:
Right now we are not really adding any metadata about a user's session log. I'm guessing that will come later?